lib/manageiq/session.rb

```diff
@@ -44,7 +44,7 @@ def self.store=(session_store)
     # Configures the session store to be used by the Rails application
     # (Vmdb::Application) and logs what is being used.
     def self.configure_session_store(adapter)
-      Vmdb::Application.config.session_store(*adapter.session_store_config_args)
+      Vmdb::Application.config.session_store(adapter.type, **args.session_options)
```

Comment:
Not sure what `args` is here...

```suggestion
      Vmdb::Application.config.session_store(adapter.type, **adapter.session_options)
```

Reply:
Looks like it was residual from the previous request for changes:

https://github.com/ManageIQ/manageiq/pull/21417#pullrequestreview-745462298

So this should be fixed.

Reply:
thanks for the assist nick